docs/accessories/developer/api_fundamentals.md

```diff
@@ -0,0 +1,44 @@
+---
+title: API Fundamentals
+project: accessories
+---
+
+### Creating an Accessory
+
+You can either create an Accessory by implementing the `Accessory` interface or by extending the `AccessoryItem` which implements the `Accessory` interface. 
+
+The first requires that you register your custom `Accessory` to the given `Item` instance using the `AccessoryRegistry#register` method while the later `AccessoryItem` will register the accessory when the class is instantiated
+
+This `Accessory` interface contains various functions that can be preformed like `tick`ing, controlling equipablity, listening to when equipment is changed, adding attribute modifiers dynamically or statically, adding custom tooltip info, and more.
+
+### Adding a Custom Accessory Renderer
+
+Creating a custom `Accessory` comes with the ability to add some rendering that is attached to the player. You can do such by creating a custom instance of `AccessoryRenderer` and registering such using the `AccessoriesRendererRegistry.registerRenderer` method. More information about the API available to developers can be found in the [rendering API section](./rendering_api.md).
```

Comment:
```suggestion
Creating a custom `Accessory` comes with the ability to add some rendering that is attached to the player. You can do so by creating a custom instance of `AccessoryRenderer` and registering it using the `AccessoriesRendererRegistry.registerRenderer` method. More information about the rendering API be found in the [rendering API section](./rendering_api.md).
```

Reply:
Yea good idea to get rid of such as it seems such is used quite often here for some such reason idk why such would such be used in instead of such other words that you used instead of such...